core/scss/components/modal.scss

```diff
@@ -329,7 +323,8 @@
     &.#{$modal__class}--default,
     &.#{$modal__class}--dialog {
       //Media Query for Mobile
-      @media #{$small-only} {
+      @media #{$small-only},
+      #{$v-small-only} {
```

Comment:
@pauljeter I know we discussed 800px, which would be `v-medium-only` but since full screen was being triggered on the small width it made a little more sense looking at the code to trigger on the small height too